examples/00-mapdl-examples/composite_dcb.py

```diff
@@ -421,6 +428,16 @@
 plt.xlabel("Displacement [mm]")
 plt.show()
 
+###############################################################################
+# Animate results using PyDPF with .anmiate() method
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Use PyDPF method .animate() to visualize the crack opening throughout the simulation as
```

Comment:
Other methods should linked using intersphinx. Add DPF to our intersphinx sources.

Reply:
I think they are already:

https://github.com/pyansys/pymapdl/blob/09d8436d407f5ebdb48352fb12d2e4dba24379da/doc/source/conf.py#L77